lib/diagnostic.js

```diff
@@ -47,7 +47,7 @@ let scopes = [/* your response here */];
 // executing it.
 //
 let fireOnAlderaan = function () {
-  // your code here
+  deathStar.fire.bind(alderaan);
```

Comment:
Close! Bind just binds the context, it doesn't actually run the function.
The answer here is to use `.call()` instead of `.bind()`. `.call()` sets the context and runs the function.
Also, you need to return the answer in order for tests to pass.